README.md

```diff
@@ -40,3 +40,41 @@ docker pull ghcr.io/input-output-hk/partner-chains/partner-chains-node:latest
 ## How to Use
 
 Refer to the [documentation](docs/user-guides) for detailed instructions on how to use the node and CLI tools.
+
+## Block Production Rewards for Validators
+
+The Partner Chains node provides a simple mechanism for exposing the mapping of block beneficiaries with produced blocks. 
+The Partner Chain Builder is responsible for using this input to accurately calculate and distribute block rewards on their partner chain, following the tokenomics they will design and implement. 
+For managing payouts, they will leverage the provided artifacts to distribute payments to validators on the partner chain ledger, applying their specific business logic.
+The process described below allows for a flexible and fair distribution of rewards, taking into account both block production and stake delegation, while ensuring that only registered participants receive rewards.
+
+| Component | Description |
+|-----------|-------------|
+| Registration | Validators must register their Cardano stake keys with Partner Chain addresses |
+| Block logging | Partner Chain nodes log block production data, including beneficiaries |
+| Reward calculation | Determined by the Partner Chain instance (e.g., N tokens per block) |
+| Distribution | Typically split between block producers and delegators based on stake |
+| Smart contracts | Track permissioned candidates, and registered candidates |
```

Comment:
There should be a new contract that holds the delegators' PC addresses

Reply:
Please remove the delegation related part.